README.md

```diff
@@ -11,12 +11,16 @@ any kind of deployment environment.
 
 ## Concourse Lite
 
-This runs outside an environment and is responsible for creating and
-destroying a VPC containing Bosh and Concourse.
+Concourse Lite is responsible for bootstrapping a full Bosh and Concourse 
+environment ([paas-cf][]). First it creates a Vagrant instance running on AWS.
```

Comment:
This link isn't quite correct in this context. paas-cf uses this Bosh and Concourse to deploy CloudFoundry.